Autopilot/AttitudeManager/Inc/AttitudeDatatypes.hpp

```diff
@@ -44,5 +44,28 @@ struct Airspeed_Data_t
     float utcTime;          
 };
 
+typedef struct {
+    float pressure, altitude, temp;
+
+    bool isDataNew; 
+    int status; //TBD but probably 0 = SUCCESS, -1 = FAIL, 1 = BUSY 
+    float utcTime; //Last time GetResult was called
+} AltimeterData_t;
```

Comment:
The name for this needs to be changed because it is the same as the one declared in altimeter.hpp

maybe use `Altimeter_Data_t`